tools/ldb_cmd.cc

```diff
@@ -2200,30 +2223,14 @@ void DBDumperCommand::DoDumpCommand() {
         fprintf(stdout, "%s ", TimeToHumanString(rawtime).c_str());
       }
       // (TODO) TTL Iterator does not support wide columns yet.
-      if (is_db_ttl_ || iter->columns().empty() ||
-          (iter->columns().size() == 1 &&
-           iter->columns().front().name() == kDefaultWideColumnName)) {
-        std::string str =
-            PrintKeyValue(iter->key().ToString(), iter->value().ToString(),
-                          is_key_hex_, is_value_hex_);
-        fprintf(stdout, "%s\n", str.c_str());
-      } else {
-        /*
-        // Sample plaintext output (first column is kDefaultWideColumnName)
-        key_1 ==> :foo attr_name1:bar attr_name2:baz
-
-        // Sample hex output (first column is kDefaultWideColumnName)
-        0x6669727374 ==> :0x68656C6C6F 0x617474725F6E616D6531:0x666F6F
-        */
-
-        std::ostringstream oss;
-        WideColumnsHelper::DumpWideColumns(iter->columns(), oss, is_value_hex_);
-        std::string str = PrintKeyValue(
-            iter->key().ToString(), oss.str().c_str(), is_key_hex_,
-            false);  // is_value_hex_ is already honored in oss. avoid
-                     // double-hexing it.
-        fprintf(stdout, "%s\n", str.c_str());
-      }
+      std::string str =
+          is_db_ttl_
+              ? PrintKeyValue(iter->key().ToString(), iter->value().ToString(),
+                              is_key_hex_, is_value_hex_)
```

Comment:
Very nit: do you think if it would make sense to go further and include `is_db_ttl_` as an argument to `PrintKeyValueOrWideColumns()`?

Reply:
I thought about this, but I felt `is_db_ttl_` context isn't much relevant inside the print function and we'd eventually have to remove the parameter when we support TTL, so I decided to leave it outside.